All Core Devs Meetings/Meeting 84.md

```diff
@@ -0,0 +1,293 @@
+# All Core Devs Meeting 84 Notes
+### Meeting Date/Time: Friday 3 April 2020, 14:00 UTC
+### Meeting Duration: 1.5 hrs
+### [Github Agenda](https://github.com/ethereum/pm/issues/162)
+### [Audio/Video of the meeting](https://www.youtube.com/watch?v=JqxVvJBhTxo)
+### [Reddit thread](https://www.reddit.com/r/ethereum/comments/fu1fzg/live_ethereum_core_devs_meeting_84_20200403_1400/)
+### Moderator: Hudson Jameson
+### Notes: Edson Ayllon
+
+---
+
+# Summary
+
+## EIP Status
+
+EIP | Status
```

Comment:
Refer the status [here](https://github.com/orgs/ethereum/projects/5) for EIPs

Reply:
b532f2f